R/build-home-index.R

```diff
@@ -189,26 +189,35 @@ sidebar_section <- function(heading, bullets, class = make_slug(heading)) {
   )
 }
 
-#' @importFrom memoise memoise
-NULL
-
-cran_link <- memoise(function(pkg) {
+cran_link <- function(pkg) {
   if (!has_internet()) {
     return(NULL)
   }
 
   cran_url <- paste0("https://cloud.r-project.org/package=", pkg)
-
-  if (!httr::http_error(cran_url)) {
+  req <- httr2::request(cran_url)
+  req <- httr2::req_cache(req, path = http_cache_dir(), max_age = 86400)
```

Comment:
what is that age? maybe add a comment?

Reply:
I just assume everyone should know how many seconds are in a day 😆

Reply:
ok ok I'll try to remember about this magic number :sweat_smile: